extensions/wikia/CreatePage/CreatePage.i18n.php

```diff
@@ -1,9 +1,8 @@
 <?php
-/** Internationalization file for /extensions/wikia/CreatePage/CreatePage extension. */
-$messages = [];
+$messages = array();
 
-$messages['en'] = [
-	'createpage-desc' => 'Allows to create a new page using Wikia\'s WYSIWYG editor',
+$messages['en'] = array(
```

Comment:
This is an outdated syntax. @rybmat can you suggest the solution for this issue, that you used the last time?